x-pack/plugins/enterprise_search/server/index_management/setup_indices.ts

```diff
@@ -98,23 +171,46 @@ const indices: IndexDefinition[] = [
     aliases: ['.elastic-connectors-sync-jobs'],
     mappings: {
       _meta: {
-        version: '1',
+        version: 1,
```

Comment:
Hmmm, I made this a string at the time to give us a little more flexibility. Not necessary?

Reply:
It's not a string in ent-search, and this was one of the things I caught in cleanup.